pkg/apis/core/types.go

```diff
@@ -2554,7 +2554,8 @@ type PodAffinityTerm struct {
 	// +optional
 	LabelSelector *metav1.LabelSelector
 	// namespaces specifies which namespaces the labelSelector applies to (matches against);
-	// null or empty list means "this pod's namespace"
+	// null or empty list means "this pod's namespace";
+	// valid regular expression patterns can be used to match multiple namespaces
```

Comment:
lets keep this field's semantics as is and add `namespaceSelector *metav1.LabelSelector` 

/cc @liggitt

Reply:
Thank you for your review. I will work on it and open a new pr.